src/components/views/settings/tabs/room/VoipRoomSettingsTab.tsx

```diff
@@ -0,0 +1,97 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useCallback, useMemo, useState } from 'react';
+import { logger } from "matrix-js-sdk/src/logger";
+import { JoinRule } from "matrix-js-sdk/src/@types/partials";
+import { EventType } from "matrix-js-sdk/src/@types/event";
+
+import { _t } from "../../../../../languageHandler";
+import { MatrixClientPeg } from "../../../../../MatrixClientPeg";
+import LabelledToggleSwitch from "../../../elements/LabelledToggleSwitch";
+import SettingsSubsection from "../../shared/SettingsSubsection";
+import SettingsTab from "../SettingsTab";
+import Modal from "../../../../../Modal";
+import ErrorDialog from "../../../dialogs/ErrorDialog";
+import { ElementCall } from "../../../../../models/Call";
+import { useRoomState } from "../../../../../hooks/useRoomState";
+
+interface ElementCallSwitchProps {
+    roomId: string;
+}
+
+const ElementCallSwitch: React.FC<ElementCallSwitchProps> = ({ roomId }) => {
+    const room = useMemo(() => MatrixClientPeg.get().getRoom(roomId), [roomId]);
+    const isPublic = useMemo(() => room.getJoinRule() === JoinRule.Public, [room]);
+    const [content, events] = useRoomState(room, useCallback((state) => {
+        const content = state?.getStateEvents(EventType.RoomPowerLevels, "")?.getContent();
+        return [content ?? {}, content?.["events"] ?? {}];
+    }, []));
+
+    const [elementCallEnabled, setElementCallEnabled] = useState<boolean>(() => {
+        return events[ElementCall.MEMBER_EVENT_TYPE.name] === 0;
+    });
+
+    const onChange = useCallback((enabled: boolean): void => {
+        setElementCallEnabled(enabled);
+
+        if (enabled) {
+            events[ElementCall.CALL_EVENT_TYPE.name] = isPublic ? 50 : 0;
+            events[ElementCall.MEMBER_EVENT_TYPE.name] = 0;
+        } else {
+            events[ElementCall.CALL_EVENT_TYPE.name] = 100;
+            events[ElementCall.MEMBER_EVENT_TYPE.name] = 100;
+        }
+
+        MatrixClientPeg.get().sendStateEvent(roomId, EventType.RoomPowerLevels, {
+            "events": events,
+            ...content,
+        }).catch(e => {
+            logger.error(e);
+
+            Modal.createDialog(ErrorDialog, {
+                title: _t('Error changing power level requirement'),
+                description: _t(
+                    "An error occurred changing the room's power level requirements. Ensure you have sufficient " +
+                    "permissions and try again.",
+                ),
+            });
+        });
+    }, [roomId, content, events, isPublic]);
+
+    return <LabelledToggleSwitch
+        data-testid="element-call-switch"
+        label={_t("Enable Element Call as an additional calling option in this room")}
+        caption={_t(
+            "Element Call is end-to-end encrypted, " +
+                    "but is currently limited to smaller numbers of users.",
```

Comment:
Please use the `brand` variable

Reply:
See https://github.com/matrix-org/matrix-react-sdk/pull/9333#issuecomment-1268864360 - the product decision was that 'Element Call' should be an immutable brand